README.md

```diff
@@ -4,4 +4,6 @@
 
 ## Installation
 
-Either clone this repo and run `make install` or place an extracted release in `$(HOME)/Library/Containers/com.luckymarmot.Paw/Data/Library/Application Support/com.luckymarmot.Paw/Extensions/`
+Either clone this repo and run `make install` or place an extracted release in your Paw extensions folder.
+
```

Comment:
I had to go via the 'place an extracted release', as make install was not working, so perhaps it is worth leaving both options

Reply:
I did that too. Not sure if I understand — I think this change still mentions both options?